utils/src/main/scala/com/salesforce/op/utils/kryo/OpKryoRegistrator.scala

```diff
@@ -67,6 +70,15 @@ class OpKryoRegistrator extends KryoRegistrator {
       classOf[GenericData.Array[_]],
       new GenericJavaCollectionSerializer(classOf[java.util.ArrayList[_]])
     )
+
+    // Streaming histogram registration
+    doClassRegistration(kryo)(
```

Comment:
hmm, we should not be doing this for `TreeMap` and `mutable.WrappedArray`, as those should be already registered with Kryo.

Reply:
This flag causes a failure if these aren't explicitly registered here:

https://github.com/salesforce/TransmogrifAI/pull/152/files#diff-36f3a0a7cf47675bc10aac982dab7fecR51

Reply:
This is weird and to my understanding should not be the case. Let's find time to look at together.